tests/pytests/test_expire.py

```diff
@@ -256,7 +258,7 @@ def test_expire_aggregate(env):
     # If not cleared, it might affect subsequent results.
     # This test ensures that the flag indicating expiration is cleared and the search result struct is ready to be re-used.
     res = conn.execute_command('FT.AGGREGATE', 'idx', '*', 'LOAD', 1, '@t')
-    env.assertEqual(res, [1, None, ['t', 'arr']])
+    env.assertEqual(res, [1, ['t', 'arr'], None])
```

Comment:
Because the expiration notification now causes reindexing of the document.
Which in turn causes the document to be moved to the end of the inverted index.